### ISSUE TITLE
Replays upside down

### ISSUE THREAD

**Issue by mad-s** [itdelatrisu/opsu#118]
When viewing replays and the "Hard Rock" mod state is not equal to it's state at the time the replay was created, the replay is upside down.

**Comment by mad-s** [comment]
There seem to be a lot of issues involving mods and replays due to getting the current value and not the stored value.

**Comment by itdelatrisu** [comment]
I'll look into this.  Part of the problem is that this isn't getting called early enough in `Game.enter(...)`:
```
			if (isReplay) {
				previousMods = GameMod.getModState();
				GameMod.loadModState(replay.mods);
			}
```
If I move that up near the top of the method, it fixes the issue with Hard Rock.  Checking out if your fix in #117 was necessary...
